src/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from './zod/makeZodErrorMap'
+export * from './zod/errorMessages'
+
+export { default as de } from './translations/de.json'
+export { default as it } from './translations/it.json'
+
+export * from './react/useZodInternationalizationWithReactIntl'
+export * from './react/ZodInternationalizationWithReactIntl'
```

Comment:
I think this will force you to install `react-intl`, otherwise it will not compile.
Maybe we could export it from `react-intl.ts`?

Reply:
Good point. I'll try it